src/managers/ApplicationCommandPermissionsManager.js

```diff
@@ -0,0 +1,410 @@
+'use strict';
+
+const { Error, TypeError } = require('../errors');
+const Collection = require('../util/Collection');
+const { ApplicationCommandPermissionTypes, APIErrors } = require('../util/Constants');
+
+/**
+ * Manages API methods for permissions of Application Commands.
+ */
+class ApplicationCommandPermissionsManager {
+  constructor(manager) {
+    /**
+     * The manager or command that this manager belongs to
+     * @type {ApplicationCommandManager|ApplicationCommand}
+     */
+    this.manager = manager;
+    /**
```

Comment:
Missing spaces here and in the fields below:
```suggestion
    this.manager = manager;

    /**
```